pytorch_lightning/trainer/optimizers.py

```diff
@@ -117,6 +123,16 @@ def configure_schedulers(self, schedulers: list, monitor: Optional[str] = None):
                         f'The "interval" key in lr scheduler dict must be "step" or "epoch"'
                         f' but is "{scheduler["interval"]}"'
                     )
+                if is_manual_optimization:
+                    invalid_keys = {'interval', 'frequency', 'reduce_on_plateau', 'monitor', 'strict'}
```

Comment:
why are reduce_on_plateau, monitor, and strict invalid?

Reply:
Because even if those keys are provided, users need to call `lr_scheduler.step(metric_to_monitor)` manually anyway since `lr_scheduler.step()` will be disabled for all learning rate schedulers including [`torch.optim.lr_scheduler.ReduceLROnPlateau`](https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.ReduceLROnPlateau) in manual optimization.
```python
scheduler = self.lr_schedulers()
scheduler.step(val_loss)
```